Gaijinizer patch/Map122.txt

```diff
@@ -1080,27 +1080,27 @@ has reached 100!#Triketos
 
 よっぽど欲求不満なのかい？
 > CONTEXT: Map122/events/5/pages/2/2/Dialogue/enemy14
-You ......... Are you frustrated?#MTLed
+Are you just sexually frustrated or something?#Ron
 > END STRING
 
 > BEGIN STRING
 なにもこんな悪魔……しかも性悪な奴に
 
 人間の一世一代の『プロポーズ』をかますなんて……
 > CONTEXT: Map122/events/5/pages/2/6/Dialogue/enemy14
-Anything like this demon ...... and a wicked man
-What a man's first generation "propose" is to be
-destroyed ...#MTLed
+Trying to surprise a demon like me, much less
+one with a horrible personality with your human
+once-in-a-lifetime marriage proposal...?#RonIffyWording
 > END STRING
```

Comment:
Think this is a bit better:
>_**Are you trying to surprise me, a demon
with a horrible personality, with your "human"
once-in-a-lifetime marriage proposal...?**_